pipelines/common.yaml

```diff
@@ -71,8 +76,34 @@ spec:
       name: build-platforms
       type: array
     - default: "false"
-      description: Enable in-development package managers, such as pre-fetch rpm packages defined in RPM lockfiles.
-      name: use-dev-package-managers
+      description: Whether to send slack notification when the pipeline run failed
+      name: send-slack-notification
+      type: string
+    - default: "release-mce-210" # Need to change for each release
```

Comment:
maybe default to `empty` to ask for making changes.

Reply:
Got it, fixed.